### ISSUE TITLE
Multi browser location/apps

### ISSUE THREAD

**Issue by SantaLaMuerte** [moom825/xeno-rat#50]
If on client side there is 2 chrome browsers, standalone or some another instance is it possible to provide "that" chrome PID or location to catch exactly "that" chrome session - at now as i see - nope.

**Comment by moom825** [comment]
it is designed to work the default chrome and default chrome path. if you want to add other browsers or browser version you are free to modify it, all that the browser cloning is just copying the data directory of the browser to another location then starting the browser defining that copied location as the data directory (in command line args). i may add in the future a mass scan of all browsers then allowing the user to select which one they want to clone in a later update. but for this moment its not going to be added, but in the future prob.

**Comment by moom825** [comment]
(didnt meant to close it)

**Comment by SantaLaMuerte** [comment]
> i may add in the future a mass scan of all browsers

yep its sound extra nice. thanks for answer, and for your work on project
